actors/abi/sector.go

```diff
@@ -94,10 +90,10 @@ type PartialTicket []byte // 32 bytes
 // TODO Porcu: refactor these types to get rid of the squishy optional fields.
 type PoStVerifyInfo struct {
 	Randomness      PoStRandomness
-	CommR           SealedSectorCID
-	Candidates      []PoStCandidate              // From OnChain*PoStVerifyInfo
-	Proofs          []PoStProof                  // From OnChain*PoStVerifyInfo
-	EligibleSectors map[SectorID]SealedSectorCID // TODO: HAMT?
+	CommR           cid.Cid
+	Candidates      []PoStCandidate      // From OnChain*PoStVerifyInfo
+	Proofs          []PoStProof          // From OnChain*PoStVerifyInfo
+	EligibleSectors map[SectorID]cid.Cid // TODO: HAMT?
```

Comment:
While we're here, what do you propose for this data structure. HAMT makes no sense here. How about a list of tuples?